src/main/resources/public/jsx/geodashreact.js

```diff
@@ -887,9 +922,9 @@ class GraphWidget extends React.Component {
                             }
                         });
                         timeseriesData = timeseriesData.sort(ref.sortData);
-
                         graphWidgetArray["widgetgraph_" + widget.id] = ref.createChart(widget.id, indexName, timeseriesData, indexName);
                         graphWidgetArray["widgetgraph_" + widget.id].id = widget.id;
+                        this.props.graphRef = ref.createChart(widget.id, indexName, timeseriesData, indexName);
```

Comment:
Do not directly mutate the props!

Reply:
How does this even work, this is not visible.  Use arrow functions and remove ref variable